llpc/context/llpcPipelineContext.cpp

```diff
@@ -377,6 +377,13 @@ void PipelineContext::setOptionsInPipeline(Pipeline *pipeline) const {
       shaderOptions.updateDescInElf = shaderInfo->options.updateDescInElf;
       shaderOptions.unrollThreshold = shaderInfo->options.unrollThreshold;
 
+      static_assert(static_cast<lgc::DenormalMode>(Vkgc::DenormalMode::Auto) == lgc::DenormalMode::Auto, "mismatch");
```

Comment:
-> Mismatch, Uppercase initial. 
Apply it to following codes.

Reply:
I am happy to change this, but every other static assertion in this file uses lowercase "mismatch".
Can I leave this change with it lowercase and then submit a follow up PR to rewrite them all to uppercase "Mismatch" if that is your preference?

Reply:
That's OK. Keep it consistent with the environment. Then change all.